vue-training/src/components/FormButton.vue

```diff
@@ -0,0 +1,38 @@
+<template lang="pug">
+  .buttons-container
+    button.form-button(
+      :class="button.class"
+      @click="$emit('handleClick')") {{ button.text }}
+</template>
+
+<script>
+export default {
+  name: 'FormButton',
+  props: {
+    button: {
+      text: String,
+      class: String
+    }
```

Comment:
```suggestion
    button: {
      type: Object,
      default: () => ({
        text: '',
        class: ''
      })
    }
```

Prop type and default value definition. See https://vuejs.org/v2/style-guide/#Prop-definitions-essential